extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/static_resource/StaticResourceBuildTimeConfig.java

```diff
@@ -0,0 +1,54 @@
+package io.quarkus.vertx.http.runtime.static_resource;
+
+import java.util.Map;
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigDocMapKey;
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.quarkus.runtime.configuration.MemorySize;
+
+@ConfigRoot(name = "static-resource", phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
+public class StaticResourceBuildTimeConfig {
+    /**
+     * The endpoint from which the static resources will be served.
+     */
+    @ConfigDocMapKey("endpoint")
+    @ConfigItem(name = ConfigItem.PARENT)
+    public Map<String, StaticResourceGeneralConfig> config;
+
+    @ConfigGroup
+    public static class StaticResourceGeneralConfig {
+        /**
+         * Indicate if the static resource endpoint is enabled
+         */
+        @ConfigItem
+        public boolean enabled;
+
+        /**
+         * Enable caching of static resource
+         */
+        @ConfigItem(defaultValue = "true")
+        public boolean enableCache;
+
+        /**
+         * The default cache size.
+         */
+        @ConfigItem(defaultValue = "10K")
+        public Optional<MemorySize> cacheSize;
```

Comment:
I think this is a number of objects, not a memory size. Could you check?

Reply:
It is indeed what you think it is. Good catch, thanks.